applications/resturant/endpoints.py

```
@@ -71,76 +94,70 @@
-                        "name": "Pacocha, Rhoda",
-                        "no_of_guests": 7,
-                        "date": "05-11-2045",
-                        "time": "10:55 PM",
-                    },
-                },
-                request_only=False,
-                response_only=True,
-            ),
-        ],
-    )
     def create(self, request, *args, **kwargs):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)